membership/index.md

```diff
@@ -0,0 +1,45 @@
+---
+redirect_from: 
+    - membership/founding-membership
+---
+
+# Become a member
+
+We’re currently looking for members to help shape the Foundation for Public Code’s long term future.
+
+Members are forward-thinking public organizations with a strong commitment to the existence of an ecosystem of public code. As a member you'll help create a global community of public code empowered organizations that pool resources and collaborate in the digital transformation of modern government. This results in higher quality services for the public that are more cost effective, with less risk and more local control.
+
+Sound interesting? Find out more [about becoming a member](https://about.publiccode.net/activities/membership-growth/) and get in touch with us!
+
+## Benefits of being a member
+
+As a member, you’ll shape the Foundation’s processes and governance.
+
+We will help you approach your digital systems in a way that promotes effective and high quality delivery, as well as:
+
+* help you work together with other public organizations - sharing software code and learnings - to reduce cost and make investments more effective
+* set up a sustainable relationship with the market - helping different (types of) companies work together without a single point of failure (e.g. service designers with developers with system maintainers)
+* make sure you and other public members are in control of their systems so you can respond to new citizen needs, policy goals or technologies
+
+Further benefits of membership include being able to:
+
+* steer the Foundation’s pipeline towards projects you’d like to reuse
+* gain recognition as a leader of systemic digital transformation
+* demonstrate return on IT investments from higher levels of administration
+* increasing the security and quality of your code through continuous auditing resources
```

Comment:
I think it would make sense to avoid any confusion over us doing 'security auditing'

Also I suspect the last three words are 'continuous auditing of resources'?

Reply:
These are remnants from the original text that I was not sure about providing a technical edit for. Could you provide alternative formulations that reflect what the codebase stewards do?

Especially the last sentence had a very long, unwieldy formulation in it, I am not sure what was meant there ad tried to reduce it to soemthing that makes sense

Reply:
tiny tweak, fixed